tensorflow/python/keras/callbacks_test.py

```diff
@@ -23,6 +23,7 @@
 import json
 import os
 import re
+import requests
```

Comment:
` File "<embedded module '_launcher'>", line 149, in run_filename_as_main
  File "<embedded module '_launcher'>", line 33, in _run_code_in_main
  File "third_party/tensorflow/python/keras/callbacks_test.py", line 26, in <module>
    import requests
ImportError: No module named requests
-- Forge runner: Test failed with exit code 1 while running`

Reply:
I added `requests` imports to  `tensorflow/tools/ci_build/install/install_pip_packages.sh` to avoid this. Does that not work? An alternative is to do a try catch around the import and not run the test if requests is not available. However that is a less preferable option as this code would not be tested.

Reply:
@mihaimaruseac any thoughts on this ?

Reply:
I might have to do some extra setup internally. Please remind me about this on Monday if I don't get to it by then

Reply:
@mihaimaruseac  friendly reminder